src/components/Header/Header.js

```diff
@@ -98,19 +101,23 @@ class Header extends React.Component {
     })
   }
   clickSearch(e) {}
+  loadStatus(status) {
```

Comment:
```suggestion
  saveStatus(status) {
```

and rename in the other places that reference it?